packages/Chem/src/analysis/activity-cliffs.ts

```diff
@@ -6,12 +6,21 @@ import {ITooltipAndPanelParams} from '@datagrok-libraries/ml/src/viewers/activit
 import {convertMolNotation, getRdKitModule} from '../package';
 import { RDMol } from '@datagrok-libraries/chem-meta/src/rdkit-api';
 import {getMCS} from '../utils/most-common-subs';
+import {BitArrayMetrics} from '@datagrok-libraries/ml/src/typed-metrics';
 
 const canvasWidth = 200;
 const canvasHeight = 100;
 
 const cashedData: DG.LruCache<String, any> = new DG.LruCache<string, string>();
 
+export type ActivityCliffsParams = {
+  molColName: string,
+  actColName: string,
```

Comment:
Don't think we need to shorten "activity" here

Reply:
Done